Source/WebCore/platform/text/PlatformLocale.h

```diff
@@ -65,7 +65,7 @@ class Locale {
         RightToLeft
     };
 
-#if PLATFORM(GTK) || PLATFORM(WPE) || PLATFORM(COCOA)
+#if !PLATFORM(WIN)
     // Returns the default writing direction for the specified locale.
     virtual WritingDirection defaultWritingDirection() const = 0;
 #endif
```

Comment:
I wonder if we should just have default implementation which returns `WritingDirection::Default` here (without ifdefs) instead.

Reply:
Implementing
```
Locale::WritingDirection Locale::defaultWritingDirection() const
{
    return WritingDirection::Default;
}
```

in C++ side, and changing this header to
```
virtual WritingDirection defaultWritingDirection() const;
```

Reply:
Sure, seems fine to me.